v3/src/components/calculator/calculator-title-bar.tsx

```diff
@@ -17,7 +17,7 @@ export const CalculatorTitleBar = observer(function CalculatorTitleBar({tile, on
       <Flex className="header-right">
         <MinimizeIcon className="component-minimize-icon" title={t("DG.Component.minimizeComponent.toolTip")}/>
         <CloseButton className="component-close-button" title={t("DG.Component.closeComponent.toolTip")}
-          onClick={()=>onCloseTile(tileId)}/>
+          onPointerDownCapture={()=>onCloseTile(tileId)}  onClick={()=>onCloseTile(tileId)}/>
```

Comment:
Presumably you don't need the `onClick` handler if you've got the `onPointerDownCapture` handler. Out of curiosity, did you try an `onPointerDown` handler?